app/controllers/v0/claims_controller.rb

```diff
@@ -9,6 +9,16 @@ def index
              each_serializer: ClaimSerializer
     end
 
+    def documents
+      params.require :file
```

Comment:
I don't really understand when to use params.require and when to just access the params.